README.md

```diff
@@ -90,15 +72,16 @@ This will also provide code coverage statistics for the various Go packages.
 
 ## Support
 
-Don’t hesitate to ask! Contact the team and community on [our support page](https://github.com/dell/karavi-observability/blob/main/docs/SUPPORT.md).
-Open an issue if you found a bug on [Github Issues](https://github.com/dell/karavi-observability/issues).
+For all your support needs or to follow the latest ongoing discussions and updates, join our Slack group. Click [Here](http://del.ly/Slack_request) to request your invite.
```

Comment:
Is the slack link to be filled in?

Reply:
We were asked to point everyone to the link above: http://del.ly/Slack_request